### DISCUSSION TITLE
Measuring up to 48 V (10 Ah) battery

### DISCUSSION THREAD

**Discussion by ibnukhak** [rlogiacco/BatterySense#21] [Q&A] [answered]
can this be used to sensor the battery above `48 V` with `10 Ah`? if which can be changed?

**Comment by rlogiacco** [comment, marked as answer]
I don't see a reason why it shouldn't work, provided you configure it properly and you use a control circuit fit for your battery.

**Comment by ibnukhak** [comment]
do I need to replace the voltage divider resistor ?, so that I get 0 - 5 volts at the point connected with Arduino, my battery is lithium polymer, sometimes the voltage is above 48 V and has 1O Ampherehour, thks before

**Comment by rlogiacco** [comment]
I have spent hours writing the project README: please read it extensively and accurately, then, if you still have doubts, I will be happy to help answering to questions that are not in the form _“give me the parameters for my battery”_
